src/views/InProgress.vue

```diff
@@ -76,42 +76,42 @@
           <div class="desktop-only">
               <ion-segment @ionChange="segmentChanged($event)" v-model="segment">
                 <ion-segment-button value="pack">
-                  <ion-label>Ready to Pack</ion-label>
+                  <ion-label>{{ $t("Ready to pack") }}</ion-label>
                 </ion-segment-button>
                 <ion-segment-button value="issue">
-                  <ion-label>Report an issue</ion-label>
+                  <ion-label>{{ $t("Report an issue") }}</ion-label>
                 </ion-segment-button>
               </ion-segment> 
               <div class="segments">
               <div v-if="segment == 'pack'">
                 <ion-item lines="none">
-                  <ion-label>Select box</ion-label>   
+                  <ion-label>{{ $t("Select box") }}</ion-label>   
                   <ion-select value="box1">
-                    <ion-select-option value="box1">Box A Type 3</ion-select-option>
-                    <ion-select-option value="box2">Box B Type 2</ion-select-option>
+                    <ion-select-option value="box1">{{ $t("Box A") }} {{ $t("Type 3") }}</ion-select-option>
+                    <ion-select-option value="box2">{{ $t("Box B") }} {{ $t("Type 2")}}</ion-select-option>
                   </ion-select>      
                 </ion-item>
               </div>
               <div v-if="segment == 'issue'">
                 <ion-item lines="none">  
-                  <ion-label>Select Issue</ion-label>  
+                  <ion-label>{{ $t("Select issue") }}</ion-label>  
```

Comment:
Need to verify if this will be internationalised

Reply:
Ok sir